react/BankInvoiceSection.tsx

```diff
@@ -50,10 +59,10 @@ const BankInvoiceSection: FC = () => {
           values={{ paymentSystemName }}
         />
       </header>
-      <div className="flex-l justify-between items-center mt6">
+      <div className={`${handles.barCodeFlexContainer} flex-l justify-between items-center mt6`}>
```

Comment:
I'd advise against using CSS terms in handles names. What if someone wants to modify this `div` to be `display: block`? We'd have a flex container which is not an actual flex container. What do you think of `barCodeContainer`?

```suggestion
      <div className={`${handles.barCodeContainer} flex-l justify-between items-center mt6`}>
```

Reply:
I get what you mean, but "flex-l justify-between items-center mt6" where already in there, I just wanted a way to reference to that particular DIV.